From: Yanhao Mo <yanhaocs@gmail.com>
Date: Mon, 15 Oct 2018 15:39:00 +0800
Subject: Remove "Mime-Type" record form timeshift-gtk.desktop

get rid of Lintian Warning: desktop-mime-but-no-exec-code
---
 src/timeshift-gtk.desktop | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/timeshift-gtk.desktop b/src/timeshift-gtk.desktop
index ed32d6f..90dfe88 100644
--- a/src/timeshift-gtk.desktop
+++ b/src/timeshift-gtk.desktop
@@ -1,6 +1,5 @@
 [Desktop Entry]
 Name=Timeshift
-MimeType=
 Exec=timeshift-launcher
 Type=Application
 GenericName=System Restore Utility
